docs/fragments/lib/android.md

```diff
@@ -1,4 +1,6 @@
-## Amplify Android (Preview)
+**Amplify Android (preview)**  
+The Amplify Android libraries are available in Java programming language. This guide shows how to build an app using our Amplify Libraries for Android (Preview) and the Amplify CLI toolchain. To use the existing AWS Mobile SDK for iOS instead, [click here](../sdk/sdk.md).
```

Comment:
Same here. The "Click here" sentence seems very gnarly. How about start the sentence with "Click here to..."

Reply:
I have kept it consistent with the message which we have on our live site today: https://aws-amplify.github.io/docs/sdk/ios/start